Kickstarter-iOS/Views/Cells/SettingsFollowCell.swift

```diff
@@ -63,6 +76,10 @@ internal final class SettingsFollowCell: UITableViewCell, ValueCell {
     self.followingSwitch.rac.on = self.viewModel.outputs.followingPrivacyOn
   }
 
+  func toggleOn(animated: Bool = true) {
+    self.followingSwitch.setOn(true, animated: animated)
```

Comment:
Why is `true` hard-coded here 🤔

Reply:
Good question. So this method could be more generic (i.e. `toggle(on: Bool, animated: Bool)`) but since we only ever need to toggle back ON we have used more specific function named `toggleOn` which implies that we're switching the toggle ON...hence `setOn(true, ...)`